game-core/src/main/java/games/strategy/debug/console/window/ConsoleWindow.java

```diff
@@ -0,0 +1,91 @@
+package games.strategy.debug.console.window;
+
+import java.awt.BorderLayout;
+import java.awt.FlowLayout;
+
+import javax.swing.JFrame;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+import javax.swing.JToolBar;
+import javax.swing.SwingConstants;
+import javax.swing.SwingUtilities;
+
+import games.strategy.ui.SwingAction;
+import games.strategy.ui.SwingComponents;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.extern.java.Log;
+import swinglib.JComboBoxBuilder;
+import swinglib.JFrameBuilder;
+import swinglib.JTextAreaBuilder;
+
+@Log
+class ConsoleWindow implements ConsoleView {
+
+  private final JTextArea textArea = JTextAreaBuilder.builder()
+      .rows(20)
+      .columns(50)
+      .build();
+
+  @Getter(AccessLevel.PACKAGE)
+  private final JFrame frame = JFrameBuilder.builder()
+      .title("TripleA Console")
+      .layout(new BorderLayout())
+      .build();
+
+
+  ConsoleWindow() {
+    final ConsoleModel model = new ConsoleModel(this);
+
+    frame.getContentPane().add(new JScrollPane(textArea), BorderLayout.CENTER);
+    frame.getContentPane().add(createButtonsToolBar(model), BorderLayout.SOUTH);
+
+    SwingUtilities.invokeLater(frame::pack);
+  }
+
+  private static JToolBar createButtonsToolBar(final ConsoleModel model) {
+    final JToolBar buttonsToolBar = new JToolBar(SwingConstants.HORIZONTAL);
+    buttonsToolBar.setFloatable(false);
+    buttonsToolBar.setLayout(new FlowLayout());
+    buttonsToolBar.add(SwingAction.of("Enumerate Threads", model::enumerateThreadsAction));
+    buttonsToolBar.add(SwingAction.of("Memory", model::memoryAction));
+    buttonsToolBar.add(SwingAction.of("Properties", model::propertiesAction));
+    buttonsToolBar.add(SwingAction.of("Copy to clipboard", model::copyToClipboardAction));
+    buttonsToolBar.add(SwingAction.of("Clear", model::clearAction));
+
+    // buttonsToolBar.add(new JLabel("Log Level:"));
```

Comment:
Shouldn't this comment get removed?

Reply:
Yes, my mistake as well to not spot this in self-review of the PR. 

removed in: f7d530948